public/apps/account/utils.tsx

```diff
@@ -43,7 +48,14 @@ export async function logout(http: HttpStart, logoutUrl?: string): Promise<void>
 export async function samlLogout(http: HttpStart): Promise<void> {
   // This will ensure tenancy is picked up from local storage in the next login.
   setShouldShowTenantPopup(null);
-  window.location.href = `${http.basePath.serverBasePath}${API_AUTH_LOGOUT}`;
+  window.location.href = `${http.basePath.serverBasePath}${SAML_AUTH_LOGOUT}`;
+}
+
+export async function openidLogout(http: HttpStart): Promise<void> {
+  // This will ensure tenancy is picked up from local storage in the next login.
+  setShouldShowTenantPopup(null);
```

Comment:
Can we segregate this into a common function like `logoutCommons` and do the following there : 
```
setShouldShowTenantPopup(null);
sessionStorage.clear();
```

It will be cleaner and promote reuse for other auth types in future.

Reply:
Addressed.